modules/namespace-exclusions.adoc

```diff
@@ -6,8 +6,39 @@
 
 :_mod-docs-content-type: CONCEPT
 [id="namespace-exclusions_{context}"]
-= Namespace exclusions
+= Namespace exclusions from admission controller enforcement
 
-By default, {product-title-short} excludes certain administrative namespaces, such as the `stackrox`, `kube-system`, and `istio-system` namespaces, from enforcement blocking. The reason for this is that some items in these namespaces must be deployed for {product-title-short} to work correctly.
+[role="_abstract"]
+By default, {rh-rhacs-first} excludes certain administrative namespaces from admission controller enforcement blocking. The reason for this is that some items in these namespaces must be deployed for {product-title-short} to work correctly. 
```

Comment:
... from the validating webhook configurations in the admission controller. Policy evaluation and enforcement is not performed on review requests originating from these administrative namespaces.